prac_05/word_occurrences.py

```diff
@@ -0,0 +1,18 @@
+"""
+CP1404 2020 - Practical 5
+Student Name: Amy Robinson
+Program - Word Occurrences
+"""
+
+word_count_dict = {}
+
+text = input("Text: ")
+words = sorted(text.split())
+
+for word in words:
+    word_count_dict[word] = word_count_dict.get(word, 0) + 1
+
+longest_word = max((len(word) for word in words))
```

Comment:
I did a test and the double brackets for max are not needed.

Reply:
Deleted, Thank you.